pkg/cluster/manager.go

```diff
@@ -406,8 +450,23 @@ func (m *Manager) Display(clusterName string, opt operator.Options) error {
 
 	// display cluster meta
 	cyan := color.New(color.FgCyan, color.Bold)
-	fmt.Printf("%s Cluster: %s\n", m.sysName, cyan.Sprint(clusterName))
-	fmt.Printf("%s Version: %s\n", m.sysName, cyan.Sprint(base.Version))
+	fmt.Printf("Cluster type:    %s\n", cyan.Sprint(m.sysName))
```

Comment:
this seems redundant for the user.

Reply:
It was `tidb Cluster: xxx` and `tidb Version: vx.y.z`